src/app/qgisapp.cpp

```diff
@@ -4271,7 +4271,25 @@ void QgisApp::updateRecentProjectPaths()
                               : QFileInfo( recentProject.path ).completeBaseName(), QDir::toNativeSeparators( recentProject.path )
                             ).replace( "&", "&&" )
                       );
-    //action->setEnabled( QFile::exists( ( recentProject.path ) ) );
+
+    QRegularExpression re_gpkg( "^(geopackage:)([^\?]+)\?(.+)$", QRegularExpression::CaseInsensitiveOption );
```

Comment:
Use camelCase for C++ code

```suggestion
    QRegularExpression reGpkg( "^(geopackage:)([^\?]+)\?(.+)$", QRegularExpression::CaseInsensitiveOption );
```